examples/connext_dds/get_publishers/c++/Foo_publisher.cxx

```diff
@@ -86,6 +86,7 @@ extern "C" int publisher_main(int domainId, int sample_count)
 {
     DDSDomainParticipant *participant = NULL;
     DDSPublisher *publisher = NULL, *publisher2 = NULL;
+    int count = 0;
```

Comment:
count is declared in the for loop below as `size_t`. We can get rid of this declaration, I think there was a fix in develop for this already.